db/data/20220705100148_backfill_hesa_updated_at_for_leading_zero_hesa_ids_on_trainees.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class BackfillHesaUpdatedAtForLeadingZeroHesaIdsOnTrainees < ActiveRecord::Migration[6.1]
+  def up
+    trainees = Trainee.imported_from_hesa.where(hesa_updated_at: nil)
+
+    trainees.find_each do |trainee|
+      hesa_trainee = Hesa::Student.find_by(hesa_id: trainee.hesa_id.to_i.to_s)
```

Comment:
Might be worth making the leading zero stripping a method so as the name can explain what's going on with the `to_i.to_s` bit.